OpenSim/Sandbox/MatlabScripts/c3d_reading.m

```diff
@@ -111,13 +106,35 @@
         rotatedData = [rotationMatrix'*vectorData']';
 
         % Write the rotated data back to the Vec3TimesSeriesTable
-        % THE BELOW LINES DO NOT WORK AND THERE IS CURRENTLY NO FIX
-        % force.updElt(0,iRow).set(0, rotatedData)
-             
+        elem = Vec3(rotatedData(1),rotatedData(2),rotatedData(3));
+        % set the value of the element
+        force.set(iRow, elem);
+
     end
 end
+%% Print the rotated markers to trc file
+
+% make trc adapter and write marker tables to file. 
+trcfileadapter = TRCFileAdapter();
+trcfileadapter.write(markers,'test_walking.trc');
+trcfileadapter.write(markers_rotated,'test_walking_rotated.trc');
 
 %% Print the force data
+% make postfix string vector for naming colomns
+postfix = StdVectorString();
+postfix.add('_x');
+postfix.add('_y');
+postfix.add('_z');
+
+% flatten the Vec3 table to a doubles table
+forces_flattened = forces.flatten(postfix);
+forces_rot_flattened = forces_rotated.flatten(postfix);
+
+% make a sto adapter and write the forces table to file.
+stofileadapater = STOFileAdapter();
```

Comment:
Is it possible to do `STOFileAdapter().write(...);`? Conceptually, you're calling a static method on STOFileAdapter.